src/Aspire.Hosting/build/Aspire.Hosting.targets

```diff
@@ -34,12 +36,23 @@
     <ItemGroup>
       <AspireProjectMetadataSource Update="@(AspireProjectMetadataSource)">
         <Source>
-        <![CDATA[namespace Projects%3B
-
+        <![CDATA[//------------------------------------------------------------------------------
+// <auto-generated>
+//     This code was generated by a tool.
+//
+//     Changes to this file may cause incorrect behavior and will be lost if
+//     the code is regenerated.
+// </auto-generated>
+//------------------------------------------------------------------------------
+
+namespace Projects%3B
+
+[global::System.CodeDom.Compiler.GeneratedCode("Aspire.Hosting", "]]>$(AspireVersion)<![CDATA[")]
```

Comment:
Let's just use `null` for the version for now and log an issue especially for it. This is going to take more work than what we are doing here.